src/app/find/mapPage/filters/FiltersModal.jsx

```diff
@@ -51,6 +52,11 @@ class FiltersModal extends Component {
             onChange={this.setFilterValues}
           />
         </div>
+        <div className="d-flex justify-content-around filterSubmitBtn">
```

Comment:
How come flex is needed for a div with only a single child? If it's meant to vertically center the button in the div, then wouldn't that happen anyway, since the spacing is just coming from padding that's equal on top and bottom?

Reply:
Fixed!